### ISSUE TITLE
Hot Reload - Obsolete

### ISSUE THREAD

**Issue by NatElkins** [slaveOftime/Fun.Blazor#69]
I notice in the docs it says Hot Reload is obsolete.  Is there an alternative?  And I am curious, is there a reason it has been marked as obsolete?

Thank you for this great library!

**Comment by albertwoo** [comment]
Yes, it was obsoleted. Because with very dotnet sdk upgrade or fsharp compiler service upgrade, it was broken again and again. Quite hard to maitain. Also it's usage is also not very user friendly. Plus, they fsharp compiler is also slow to compile for files marked as hot reload. Yeah, quiker than full rebuild, but still slow for a large projects. So I did not want to maintain it. In my own projects, I just rebuild everytime.

There is no alternative. Just hope fsharp core team can came up some hot reload solution like csharp has. Yeah, disapointed, because it may take years for that to come.

[Check the fsharp hot reload request](https://github.com/dotnet/fsharp/issues/11636)
